src/form-faker.js

```diff
@@ -1,11 +1,35 @@
 function processForm(userMappings, options) {
   console.log('form-faker: faking data...');
+  options = options || {};
   const dictionary = getDictionary();
   var inputs = document.getElementsByTagName('input');
 
   const gender = faker.random.number(1);  // 0=female, 1=male; for faker's parameters (https://github.com/Marak/faker.js/blob/master/lib/name.js#L21)
-  const firstName = faker.name.firstName(gender);
-  const lastName = faker.name.lastName();
+  let firstName;
+  let lastName;
+  if (options.nameGenerator === 'marvel') {
+    const names = marvelCharacterNames[faker.random.number(1252)].split(' ');
```

Comment:
Pleas use `marvelCharacterNames.length` for random number parameter so that future updates of character list don't require updating constant value.